src/App.tsx

```diff
@@ -35,26 +37,28 @@ export default function () {
     <Router>
       <div className={pageContainer}>
         <Navbar />
-        <div className={bodyContainer}>
-          <Switch>
-            <Route path="/">
-              <LazyComponent lazyImported={<Landing />} />
-            </Route>
-            <Route path="/cast">
-              <LazyComponent lazyImported={<Cast />} />
-            </Route>
-            <Route path="/terms">
-              <Terms />
-            </Route>
-            <Route path="/privacy">
-              <Privacy />
-            </Route>
-            <Route path="/wallet">
-              <BurnerWallet />
-            </Route>
-          </Switch>
+        <div className={overflowEffects}>
```

Comment:
Bg blur will overflow the container because of the moving (especially on mobiles), so there's no way to avoid this